### PR TITLE
Wet ants

### PR DESCRIPTION
**Pull request by Eggpielord**
[FEATURE]

## About the PR 
This adds a new wet ants chemical consisting of water and ants that prevents slippage. 

Example can be seen here;
https://streamable.com/itpp40

## Why's this needed? 
If this chemical is the most present one in a puddle, you can't slip on it. Creating an anti-slip chemical for space maps. It also applies the touch effect of regular ants upon contact, using up a portion of the chemical in the process. 

Full credit goes to zjdtmkhzt for bringing my dream to life by providing the code.

## Changelog

```
(u) Egg
(+) Added new "wet ants" chem that prevents you from slipping on puddles that have it as most common reagent.

### CODE REVIEW

**Comment by pali6** [comment]
Do the ants in the water hold your feet / shoes and that prevents you from slipping? 🤔
On a (very slightly) more serious note I feel like the idea of there being a "wet ants" reagent sorta fits into Goonstation-brand silliness. However, this effect seems pretty meh but also no idea what an appropriate effect would be.

**Comment by Eggpielord** [comment]
It was a chem thrown together in a couple of minutes. Any suggestions for an actually practical use would be great.

File: code/modules/chemistry/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -3783,6 +3783,14 @@ datum
...
 			mix_phrase = "The ants arachnify. What?"
 
+		wet_ants
+			name = "wet ants"
+			id = "wet_ants"
+			result = "wet_ants"
+			required_reagents = list("ants" = 1, "water" = 1)
+			result_amount = 2
+			mix_phrase = "The ants mix into the water, forming a suspension."
+
```
Comment (Tarmunora) [inline code/modules/chemistry/Chemistry-Recipes.dm:3793]: making it only react in a puddle would be funny

**Review by Tarmunora — APPROVED** [review]

**Comment by DimWhat** [comment]
So apparently this chem has the whole "get hurt on touch" thing for ants, which is cool and all but it appears to ignore shoes and apply the damage regardless.

**Pull request closed without merging**